prometheus/promhttp/http.go

```diff
@@ -370,7 +375,12 @@ type HandlerOpts struct {
 	// a trailing ".0" if they would otherwise look like integer numbers
 	// (which changes the identity of the resulting series on the Prometheus
 	// server).
```

Comment:
Let's remove all the commentary above and leave only Deprecated note. Users can navigate to that field for the description.